packages/fxa-settings/src/components/ConfirmWithLink/index.stories.tsx

```diff
@@ -0,0 +1,36 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import ConfirmWithLink from '.';
+import AppLayout from '../../components/AppLayout';
+import { LocationProvider } from '@reach/router';
+import { Meta } from '@storybook/react';
+import { Subject } from './mocks';
+
+export default {
+  title: 'components/ConfirmWithLink',
+  component: ConfirmWithLink,
+} as Meta;
+
+const storyWithProps = ({ ...props }) => {
+  const story = () => (
+    <LocationProvider>
+      <AppLayout>
+        <Subject {...props} />
+      </AppLayout>
+    </LocationProvider>
+  );
+  return story;
+};
+
+export const Default = storyWithProps({});
+
+export const UserCanGoBack = storyWithProps({
+  canGoBack: true,
+});
+
+export const OpenWebmailButtonIsVisible = storyWithProps({
+  isOpenWebmailButtonVisible: true,
+});
```

Comment:
It could be nice to see this component with a different header and text to show those pieces are dynamic.

Reply:
We don't have defaults set for these strings, so these are already custom values passed in to the story.